src/componets/Person/Person.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Person({ name, age, sex, isMarried, partnerName }) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {name}
+      </h2>
+      {age ? (
+        <p className="Person__age">
+          I am
+          {' '}
+          {age}
+        </p>
+      ) : (
+        null
+      )}
```

Comment:
```suggestion
      {age && (
        <p className="Person__age">
          I am
          {' '}
          {age}
        </p>
      )}
```
Давай використовувати && при conditional rendering